internal/impl/otlp/tracer_otlp.go

```diff
@@ -19,27 +19,37 @@ import (
 	"github.com/benthosdev/benthos/v4/public/service"
 )
 
-func init() {
-	spec := service.NewConfigSpec().
-		Summary("Send tracing events to an [Open Telemetry collector](https://opentelemetry.io/docs/collector/).").
-		Field(service.NewObjectListField("http",
-			service.NewURLField("url").
-				Description("The URL of a collector to send tracing events to.").
-				Default("localhost:4318"),
-		).Description("A list of http collectors.")).
-		Field(service.NewObjectListField("grpc",
-			service.NewURLField("url").
-				Description("The URL of a collector to send tracing events to.").
-				Default("localhost:4317"),
-		).Description("A list of grpc collectors.")).
-		Field(service.NewStringMapField("tags").
-			Description("A map of tags to add to all tracing spans.").
-			Default(map[string]string{}).
-			Advanced())
+const (
+	svcNameKey = semconv.ServiceNameKey
+	svcVerKey  = semconv.ServiceVersionKey
+)
 
+var cliVersion = cli.Version
```

Comment:
Could this cause any bugs during the build/ testing ?
can I reference this variable this way ?